src/assets/i18n/en.json5

```diff
@@ -4019,8 +4019,12 @@
 
 
 
+  "workspace.search.results.head": "Your submissions",
+
   "workflowAdmin.search.results.head": "Administer Workflow",
 
+  "workflow.search.results.head": "All tasks",
```

Comment:
I'd like to recommend we change the text of this label to say "Workflow tasks".  I've always found "All tasks" to be a confusing name, as it's unclear what it refers to. 

So, this PR might be a good opportunity to just change the wording here, as well as in the `mydspace.show.workflow` i18n key.

Reply:
done